Update conveners for 2021

graeme-a-stewart opened the pull request:
Try to make the links to email the conveners roughly
standard, with one "mailto:" to all (though some groups
maintain separate convener email addresses).

eduardo-rodrigues commented on _workinggroups/dataanalysis.md:
```diff
@@ -24,9 +24,9 @@ well-defined set of problems.
 
 Former Convenors:
-- Danilo Piparo (ROOT), Andrea Rizzi (CMS)
+- Danilo Piparo (ROOT), Andrea Rizzi (CMS), Paul Laycock (Belle II),
```
```suggestion
- Danilo Piparo (ROOT), Andrea Rizzi (CMS), Paul Laycock (Belle II)
```

eduardo-rodrigues replied:
Aside question: is that too much of a pain to add time frames to convenerships as we did for PyHEP, see the bottom of https://hepsoftwarefoundation.org/workinggroups/pyhep.html?

eduardo-rodrigues commented on _workinggroups/detsim.md:
```diff
@@ -19,9 +19,10 @@ technologies) are an important area of work.
 discussions are held on the [hsf-simulation](https://groups.google.com/forum/#%21forum/hsf-simulation) list.
 
-Convenors: Gloria Corti (LHCb) <mailto:Gloria.Corti@cern.ch>, Philippe Canal (CMS) <mailto:philippe.canal@cern.ch>, Witek Pokorski (Geant4) <mailto:Witold.Pokorski@cern.ch>
+- Convenors: Ben Morgan (ATLAS, Geant4), Kevin Pedro (CMS), Krzysztof Genser (Mu2e+Geant4)
```
```suggestion
- Convenors: Ben Morgan (ATLAS, Geant4), Kevin Pedro (CMS), Krzysztof Genser (Mu2e, Geant4)
```

eduardo-rodrigues replied:
Suggestion for consistency with Ben's way of providing affiliations.

eduardo-rodrigues commented on _workinggroups/toolsandpackaging.md:
```diff
@@ -56,15 +56,24 @@ rapidly evaluate each tool for features, ease of use and flexibility.
-- Graeme A Stewart (CERN)
+### Former Conveners
+
+- Bendikt Hegner (CMS, EP-SFT)
```
```suggestion
- Benedikt Hegner (CMS, EP-SFT)
```

eduardo-rodrigues commented on _workinggroups/toolsandpackaging.md:
```diff
@@ -56,15 +56,24 @@ rapidly evaluate each tool for features, ease of use and flexibility.
+- Bendikt Hegner (CMS, EP-SFT)
+- Elizabeth Sexton-Kennedy (CMS)
+- Graeme A Stewart (ATLAS)
+- Ben Morgan (ATLAS and superNEMO)
```
Ben is mentioned as ATLAS and Geant4 above. @drbenmorgan, what  exactly are your affiliations?

eduardo-rodrigues reviewed (approved):
I made some tiny suggestions. Let me pre-approve anyway.

graeme-a-stewart commented:
> I made some tiny suggestions. Let me pre-approve anyway.

Thanks @eduardo-rodrigues, fixed the small points you found and implemented "terms of office" for all of the previous conveners.

klieret reviewed (approved):
LGTM!
